src/cmd/sanity_test/sanityTest.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"reflect"
+	"strings"
+
+	"github.com/alcionai/corso/src/internal/connector/graph"
+	msgraphsdk "github.com/microsoftgraph/msgraph-sdk-go"
+	"github.com/microsoftgraph/msgraph-sdk-go/models"
+)
+
+func main() {
+	adapter, err := graph.CreateAdapter(
+		os.Getenv("AZURE_TENANT_ID"),
+		os.Getenv("AZURE_CLIENT_ID"),
+		os.Getenv("AZURE_CLIENT_SECRET"),
+	)
+	if err != nil {
+		os.Exit(1)
+		return
+	}
+
+	testUser := os.Getenv("CORSO_M365_TEST_USER_ID")
+	folder := strings.TrimSpace(os.Getenv("RESTORE_FOLDER"))
+
+	client := msgraphsdk.NewGraphServiceClient(adapter)
+
+	if os.Getenv("EXCHANGE_TEST") == "true" {
```

Comment:
As mentioned about, this should ideally turn into a switch with one branch for each service.